### PR TITLE
add crossplane & cloud operator installation to cs-operator

### PR DESCRIPTION
**Pull request by ZhuoxiLi**
Using the following CS CR for installing the operators, tested on Fyre:
```
spec:
  features:
    bedrockshim:
      enabled: true
```

### CODE REVIEW

**Comment by pgodowski** [comment]
@arturobrzut pls work with @ZhuoxiLi  on finalizing this. Thank you both!

**Comment by ZhuoxiLi** [comment]
@horis233 @Daniel-Fan @arturobrzut this pr is ready to review, thanks!

File: controllers/constant/crossplane.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+metadata:
+  name: ibm-crossplane-operator-app
+  namespace: {{ .MasterNs }}
+spec:
+  channel: {{ .Channel }}
+  installPlanApproval: Automatic
+  name: ibm-crossplane-operator-app
+  source: {{ .CatalogSourceName }}
+  sourceNamespace: {{ .CatalogSourceNs }}
+  startingCSV: ibm-crossplane-operator.v1.0.0
```
Comment (horis233) [inline controllers/constant/crossplane.go:31]: We can remove this `startingCSV`

Comment (ZhuoxiLi) [reply controllers/constant/crossplane.go]: removed, please review again

**Comment by horis233** [comment]
@ZhuoxiLi 
Would you mind providing an example CR in the PR?

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**